target/linux/ipq40xx/image/Makefile

```diff
@@ -32,6 +32,19 @@ define Device/UbiFit
 endef
 
 
+define Device/8dev_jalapeno
+	$(call Device/FitImage)
+	$(call Device/UbiFit)
+	DEVICE_DTS := qcom-ipq4018-jalapeno
+	KERNEL_INSTALL := 1
```

Comment:
I don't see an explanation why the raw kernel is required as extra file. Copy/Paste leftover?

Reply:
Yes, It was copied straight from 8devices branch.